build/tasks/minify.js

```diff
@@ -28,8 +28,9 @@ module.exports = async function minify( { filename, dir, esm } ) {
 					" | (c) OpenJS Foundation and other contributors" +
 					" | jquery.org/license */\n"
 			},
-			mangle: true,
 			inlineSourcesContent: false,
+			mangle: true,
+			module: esm,
```

Comment:
Where is this option coming from? I don't see anything in the docs: module, Ignored. Currently, all files are treated as module.

Reply:
This was required to get the latest version working. Open the type declarations and see `TerserCompressOptions` in the [minify function](https://swc.rs/docs/usage/core#minify)